src/index.html

```diff
@@ -1,12 +1,510 @@
 <!doctype html>
 <html lang="en">
+
   <head>
+    <title>Catalog</title>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <link rel="stylesheet" href="./styles/header.scss">
+    <link rel="stylesheet" href="./styles/card.scss">
+    <link rel="stylesheet" href="./styles/stars.scss">
+    <link rel="stylesheet" href="./styles/catalog.scss">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <!-- #region nav -->
+    <header class="nav">
```

Comment:
```suggestion
    <header class="nav">
```
why header has a nav classname?